3ds2/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="checkout_3ds2_da_registration_title">Safe and swift checkout!</string>
```

Comment:
Do we have translations for these?

Reply:
We don't have translations yet. Can we request them?